src/frequenz/client/common/v1alpha8/pagination/__init__.py

```diff
@@ -0,0 +1,91 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Module to define the pagination used with the common client."""
+
+from __future__ import annotations  # required for constructor type hinting
+
+from dataclasses import dataclass
+from typing import Self
+
+# pylint: disable=no-name-in-module
+from frequenz.api.common.v1alpha8.pagination.pagination_info_pb2 import (
+    PaginationInfo,
+)
+from frequenz.api.common.v1alpha8.pagination.pagination_params_pb2 import (
+    PaginationParams,
+)
+
+# pylint: enable=no-name-in-module
+
+
+@dataclass(frozen=True, kw_only=True)
+class Params:
```

Comment:
better keep the original name, no?  This could be ambiguous.

```suggestion
class PaginationParams:

Reply:
I copied it from v1 (non-versioned in this repo) like that. But true, we can use this opportunity to rename it in alpha8